index.js

```diff
@@ -27,6 +27,9 @@ class ServerlessSecretBaker {
 
   async writeEnvironmentSecretToFile() {
     const providerSecrets = this.serverless.service.provider.environmentSecrets;
+    if (!providerSecrets) {
+      return;
```

Comment:
Don't know if my above comment is a real issue, but if so, this would address it.

```suggestion
      providerSecrets = {};
```

Reply:
Oh, I see you meant if not having the file is an issue, no there was no deploy issue without the file. I kind of think that not including the file at all is preferable, but I'm not stuck on it.